core/jobs.py

```diff
@@ -1193,7 +1450,8 @@ def get_continuous_computations_info(cc_classes):
 
 def get_stuck_jobs(recency_msecs):
     """Returns a list of jobs which were last updated at most recency_msecs
-    milliseconds ago and have experienced more than one retry."""
+    milliseconds ago and have experienced more than one retry.
+    """
```

Comment:
"Returns" docstring?

The stuff in JobCleanupManager below also needs documenting.